### ISSUE TITLE
"Get Started" button 

### ISSUE THREAD

**Issue by sophialinn** [state-hiu/2C-GeoNode#218]
Currently links to generic GeoNode User Guide info as opposed to the 2C GeoNode custom guide that we had created. @d3netxer should have the latest version

**Comment by kalxas** [comment]
@d3netxer can you please point us to the latest version of this document? I guess it has to be migrated from the old source code, right?

**Comment by d3netxer** [comment]
yeah, it needs to be migrated. The files can be found here: https://github.com/state-hiu/2C-Geonode-Templates
